tests/e2e/nightly/models/test_qwen3_32b_int8.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2025 Huawei Technologies Co., Ltd. All Rights Reserved.
```

Comment:
Please remove this file, and add this file with job enable later.

[1] https://github.com/vllm-project/vllm-ascend/blob/main/.github/workflows/vllm_ascend_test_nightly.yaml#L46